Example/PrefireExampleTests/PrefireExampleTests.swift

```diff
@@ -3,4 +3,6 @@ import Prefire
 
 // MARK: - Generated Tests
 
-let tests = PreviewTests.self
+let tests = _TestFile.self
```

Comment:
```suggestion
private let tests = _TestFile.self
```


I get an error when running the tests, but adding `private let` fixes it.

<img width="841" height="130" alt="Image" src="https://github.com/user-attachments/assets/b0a60353-5461-467b-9522-146b7ee9fe33" />